docs/contributing/documentation-guidelines/index.md

```diff
@@ -1,5 +1,7 @@
 # Documentation guidelines
 
+## Contribution guidelines on Autoware Documentation
```

Comment:
What made you add this section? I feel it's a bit redundant.

Reply:
As I am going to add a new section below, I felt that the table of contents of this page becomes weird without the section name on the part that currently exists. 

With this modification:
![image](https://user-images.githubusercontent.com/44218668/192910391-0e4de612-b2ca-4982-a48f-cca01fa21065.png)


Without:
![Uploading image.png…]()

Reply:
> With this modification:

Hmm, I still feel weird even with this modification applied.

How about this?

```md
## Workflow

Contributions to Autoware's documentation are welcome ...

## Style guide

You should refer to the ...

## Tips

### How to preview your modification
```

Reply:
Sorry for the late reply. I will take that idea, thank you!
https://github.com/autowarefoundation/autoware-documentation/pull/203/commits/71cdf56f0afd5c453dcf54d5986d1dc142e54fe6